sys-devel/llvm/llvm-18.0.0.9999.ebuild

```diff
@@ -65,18 +80,28 @@ BDEPEND="
 RDEPEND="
 	${RDEPEND}
 	!sys-devel/llvm:0
+	llvm_projects_clang? ( ${PYTHON_DEPS} )
 "
 PDEPEND="
 	sys-devel/llvm-common
 	sys-devel/llvm-toolchain-symlinks:${LLVM_MAJOR}
 	binutils-plugin? ( >=sys-devel/llvmgold-${LLVM_MAJOR} )
+	llvm_projects_clang? (
+		>=sys-devel/clang-common-${PV}
+		~sys-devel/clang-runtime-${PV}
+		sys-devel/clang-toolchain-symlinks:${LLVM_MAJOR}
+	)
 "
 
-LLVM_COMPONENTS=( llvm cmake third-party )
+LLVM_COMPONENTS=( llvm clang clang-tools-extra cmake compiler-rt libunwind/include/mach-o lld runtimes third-party )
 LLVM_MANPAGES=1
 LLVM_USE_TARGETS=provide
 llvm.org_set_globals
 
+pkg_setup() {
```

Comment:
Do we need this explicitly? From a quick skim, python-single-r1 is the only inherited eclass providing pkg_setup so it'll get called anyway